priv/ie_gen_v1.erl

```diff
@@ -1718,7 +1721,8 @@ msgs() ->
 	 {"Redundant Transmission Parameters",	   'C', {'-', '-', '-', 'X'},
 	    [{"F-TEID",				   'M', {'-', '-', '-', 'X'}},
 	     {"Network Instance",		   'O', {'-', '-', '-', 'X'}}]},
-	 {"BAR ID",				   'C', {'X', '-', '-', 'X'}}]},
+	 {"BAR ID",				   'C', {'X', '-', '-', 'X'}},
+	 {"IPFIX Policy",		   'O', {'X', 'X', 'X', 'X'}}]},
```

Comment:
Is this format documented somewhere, @RoadRunnr ? @fpesti could you please add some comments on the file itself about the contents of the tuple otherwise?

Reply:
The format is a direct translations of IE presence tables in TS 29.244, clause 7

Reply:
Adding that as a comment would be a minimum ^_^

Reply:
reference added